src/prefect/results.py

```diff
@@ -369,9 +369,10 @@ async def create(
         obj: R,
         cache_object: bool = True,
     ) -> "UnpersistedResult[R]":
+        description = f"Unpersisted result of type `{type(obj).__name__!s}`"
```

Comment:
We'll want to do this for persisted results too. `!s` is the default btw, we can omit that.

Reply:
TIL!

Reply:
Good callout @madkinsz, I hadn't looked at that class. Updated that and also added a markdown link to the description instead of just the bare url